src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -125,10 +130,60 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         return Response.ok(body).build();
     }
 
+    @GET
+    @Path("{instanceName}/admin")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAdminList(@PathParam("instanceName") String instanceName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
+        List<String> instanceGithubTeams = instance.getSpec().getGithub().getTeams();
+
+        List<Object> instanceTeamMembers = new ArrayList<>();
+        List<User> allInstanceAdmins = new ArrayList<>();
+
+        TeamService teamService = new TeamService(client);
+
+        // Loop through each team listed in the org defined by the kabanero instance CRD
+        for (Team orgTeam : teamService.getTeams(instanceGithubOrg)) {
+            List<User> teamMembers = new ArrayList<>();
+            JsonObject teamObject = new JsonObject();
+            teamObject.addProperty("name", orgTeam.getName());
+            teamObject.addProperty("id", orgTeam.getId());
+            // Loop through each team and get all members that belong to that team
+            for (User member : teamService.getMembers(orgTeam.getId())) {
+                User githubUser = new UserService(client).getUser(member.getLogin());
+                teamMembers.add(githubUser);
+                // Loop through all the teams defined in the kabanero instance CRD and make sure the team retruned from the github api is listed in the kabanero instance CRD teams
+                for (String instanceAdminTeam : instanceGithubTeams) {
+                    if (instanceAdminTeam.equals(orgTeam.getName()) && !allInstanceAdmins.stream().anyMatch(admin -> member.getLogin().equals(admin.getLogin()))) {
+                        allInstanceAdmins.add(githubUser);
+                    }
+                }
+            }
+            teamObject.add("members", new Gson().toJsonTree(teamMembers, new TypeToken<List<User>>() {}.getType()));
+            instanceTeamMembers.add(teamObject);
```

Comment:
To simplify things and break down many http requests for each user we should:

1. loop each team in the configured GitHub org
1. If orgTeam name match against one of the names in the instanceGitHub teams (configured admin teams)
1. then add those members to a function scoped var `admins`
1. let the loop continue and return `admins` var
1. Frontend puts admin usernames in table
1. Frontend sends async request for each admin to new endpoint `"{instanceName}/user"` and fills in data as it comes back.

This will prevent us from having another request for every admin to get more profile data (will help a lot of there are 50+ admins for example) this request can get looooooooong

so then we will create a separate endpoint `"{instanceName}/user"` to get info on the user. This way the frontend can call this endpoint for each admin after they call this endpoint. That will allow each request to be called asynchronously and speed up request time dramatically instead of blocking for each admin in this endpoint.